verticapy/tests/vDataFrame/test_vDF_combine_join_sort.py

```diff
@@ -234,7 +234,6 @@ def test_vDF_pivot(self, amazon_vd):
         assert amazon_pivot.shape() == (239, 28)
         assert amazon_pivot["pv_Acre"].count() == 239
 
-    @pytest.mark.xfail(reason = "Wrong result")
```

Comment:
I wonder why it shows up here! This change was in your previous PR. It might be because you have used the same branch!